calico/felix/frules.py

```diff
@@ -261,7 +236,7 @@ def install_global_rules(config, v4_filter_updater, v6_filter_updater,
             },
             {
                 CHAIN_FORWARD: forward_deps,
-                CHAIN_INPUT: input_deps,
+                CHAIN_INPUT: input_deps
```

Comment:
Unlike Javascript, trailing commas are a good thing in Python, please put them back.  (They're good because they reduce the size of diffs and conflicts when you add a new item and the reduce the possibility for missing a comma when rearranging a list.)